config/settings/base.py

```diff
@@ -295,7 +299,13 @@ def env(name, default=NoDefaultValue, type_=str):
 # WHITENOISE_ROOT = PROJECT_ROOT.joinpath(static_dir_root)
 
 SOCIALACCOUNT_PROVIDERS = {
-    "github": {"SCOPE": ["read:user", "user:email", "repo", "read:org"]},
+    "github": {
+        "SCOPE": env(
+            "GITHUB_OAUTH_SCOPES",
+            default="read:user,user:email,public_repo",
```

Comment:
Without the `read:org` scope, this currently throws an exception:

```
ForbiddenError: 403 You need at least read:org scope or user scope to list your organizations.
```

So if that's actually a use-case we want to support (that is, only showing public repos owned directly by the logged-in-user), we need to make some changes to `get_all_org_repos` in `gh.py` to handle that.

Reply:
Is it possible we could be using this API call instead?
https://developer.github.com/v3/repos/#list-your-repositories

Ideally, we shouldn't need users of MetaShare to grant access to every Github organization they're a member of.  We just need to know which repositories they're authorized to contribute to.

Reply:
D'oh -- yes, that works. Early on we were only seeing user-owned repos until we added in an `organizations` query, but apparently that had to do with other bugs which we since fixed. This works fine if we remove the specific query of the user's orgs. 👍